packages/cli/__tests__/daemon/__unit__/DaemonDecider.unit.test.ts

```diff
@@ -125,12 +131,195 @@ describe("DaemonDecider tests", () => {
 
         const daemonDecider = new DaemonDecider(["anything"]);
 
-        const testPort = "1234";
-
+        const testSocket = "/fake/pipe/path";
         (daemonDecider as any).mParms = ["one", "two", "--daemon"];
-        process.env.ZOWE_DAEMON = testPort;
+        process.env.ZOWE_DAEMON = testSocket;
         (daemonDecider as any).initialParse();
-        expect((daemonDecider as any).mPort).toBe(parseInt(testPort, 10));
+        expect((daemonDecider as any).mSocket).toBe(process.platform === "win32" ? "\\\\.\\pipe\\" + testSocket : testSocket);
+
+    });
+
+    (process.platform !== "win32" ? it : it.skip)("should try to delete an existing socket on *nix", () => {
```

Comment:
Would it be ok to group these into a `(process.platform !== "win32" ? describe : describe.skip)` ?
Not a required change though 😋